service-raycluster-autoscaler-head-svc.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
```

Comment:
looks like dev output - should this be removed?

Reply:
removed